WORKSPACE

```diff
@@ -853,6 +853,14 @@ load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
 
 grpc_deps()
 
+load("@upb//bazel:workspace_deps.bzl", "upb_deps")
```

Comment:
Is this a newly required dependency from `@com_github_grpc_grpc`?

Reply:
Yep. It’s added by `grpc_deps`. Same for the apple stuff.